src/modules/samples/rust/src/lib.rs

```diff
@@ -1,24 +1,26 @@
 // Copyright (c) Microsoft Corporation. All rights reserved..
 // Licensed under the MIT License.
 
+use sample::Sample;
 use std::ffi::CStr;
 use std::ffi::CString;
 use std::os::raw::{c_char, c_int, c_uint, c_void};
+use std::ptr;
```

Comment:
What is the convention for ordering `use`? More specifically is there a preference for where `use sample::Sample;` should be placed (before/after `mod sample`)?